installer/templates/phx_umbrella/apps/app_name_web/config/config.exs

```diff
@@ -15,7 +15,10 @@ config :<%= web_app_name %>, <%= endpoint_module %>,
   url: [host: "localhost"],
   secret_key_base: "<%= secret_key_base %>",
   render_errors: [view: <%= web_namespace %>.ErrorView, accepts: ~w(<%= if html do %>html <% end %>json), layout: false],
-  pubsub: [name: <%= web_namespace %>.PubSub, adapter: Phoenix.PubSub.PG2]
+  pubsub: [name: <%= web_namespace %>.PubSub, adapter: Phoenix.PubSub.PG2]<%= if live do %>,
+  live_view: [
+    signing_salt: "<%= live_view_signing_salt %>"
+  ]<% end %>
```

Comment:
There was a suggestion to remove the separate live_view signing_salt.  That will require more coordination between the two projects (and clarification that it's OK to proceed with that route).  For now, I have the code generated based [on the instructions](https://github.com/phoenixframework/phoenix_live_view)